arcade/camera.py

```diff
@@ -384,7 +384,7 @@ def scale(self) -> Tuple[float, float]:
         return self._scale
 
     @scale.setter
-    def scale(self, new_scale: Tuple[int, int]) -> None:
+    def scale(self, new_scale: Tuple[int | float, int | float]) -> None:
```

Comment:
The | operator requires Python 3.10
Arcade still supports 3.8, so we should use Usion.

Still wonder if float would be enough.

Reply:
Hmm you might be right, due to the "numeric tower."  I'll check